util.py

```diff
@@ -418,7 +432,8 @@ def getSpacingDims(coordinates, warn=True, shapeTolerance=0.01, spacingTolerance
         # This is 5 in the case of the second dimension
         # TODO May want to consider taking an average of all values in case there are small differences?
         # Not sure I see a direct need to do this now
-        numNZIndices = nzIndices[0] + 1
+        #numNZIndices = nzIndices[0] + 1
+        numNZIndices = 1
```

Comment:
I'm not quite sure this is the proper thing to do.

Can you explain what your reasoning for this was?